app/import.go

```diff
@@ -86,6 +100,12 @@ func ImportLine(line LineImportData, dryRun bool) *model.AppError {
 		} else {
 			return ImportChannel(line.Channel, dryRun)
 		}
+	case line.Type == "user":
+		if line.User == nil {
+			return model.NewLocAppError("BulkImport", "app.import.import_line.null_user.error", nil, "")
```

Comment:
Minor but mind using the new `NewAppError()` function and setting the status code? I know this is mainly used from the CLI so the status code doesn't make much sense but I think it's better to just it everywhere unless there are any objections